src/redux/books/booksSlice.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-param-reassign */
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import axios from 'axios';
 import API_URL from '../../globals';
```

Comment:
Good job working on the project requirements.While to run your project I encoutered an error because of this: 

![Screenshot 2023-06-24 at 2 15 17 AM](https://github.com/kazaneza/Bookstore/assets/79506105/fb0bbe4e-df1b-477d-b795-9bcf6bbb5e5d)

![Screenshot 2023-06-24 at 2 18 45 AM](https://github.com/kazaneza/Bookstore/assets/79506105/ae79efb3-0879-4534-9186-52c342b33f13)


Please kindly fix this issue